onnxruntime/test/platform/apple/generate_ipa_export_options_plist.py

```diff
@@ -0,0 +1,54 @@
+import argparse
+
+plist_file_content = """
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>method</key>
+    <string>development</string>
+    <key>teamID</key>
+    <string>{team_id}</string>
+    <key>provisioningProfiles</key>
+    <dict>
+        <key>ai.onnxruntime.tests.ios-package-test</key>
+        <string>{provisioning_profile_uuid}</string>
+    </dict>
+    <key>signingStyle</key>
+    <string>manual</string>
+</dict>
+</plist>
+"""
+if __name__ == "__main__":
+    # handle cli args
+    parser = argparse.ArgumentParser(
+        "Generates a PList file to the relevant destination. This PList file contains the properties to allow a user to generate an IPA file for the ios-package-test. "
+    )
+
+    parser.add_argument("--dest_file", type=str, help="Path to output the PList file to.", required=True)
+    parser.add_argument(
+        "--apple_team_id",
+        type=str,
+        help="The Team ID associated with the provisioning profile. You should be able to find this from the Apple developer portal under Membership.",
+        required=True,
+    )
+    parser.add_argument(
+        "--provisioning_profile_uuid",
+        type=str,
+        help="The Provisioning Profile UUID, which is the filename of the installed provisioning profile without the .mobileprovision extension. ",
```

Comment:
```suggestion
        help="The Provisioning Profile UUID. ",
```

I think it's not necessarily part of the filename. however, there is a UUID field within the mobileprovision file itself. not sure how much detail we need to go into about it in the help text.

Reply:
Oh, I didn't know that the UUID could be found in the mobileprovision file -- I updated to specify looking in there